### PR TITLE
Switch to 6.docs.plone.org (was 6.dev-docs.plone.org)

### PR DESCRIPTION
**Pull request by ksuess**

### CODE REVIEW

**Comment by ksuess** [comment]
One task of https://github.com/plone/documentation/issues/1140

File: news/497.feature [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Switch to 6.docs.plone.org (was 6.dev-docs.plone.org)
```
Comment (stevepiercy) [inline news/497.feature:1]: I would argue that this is not a new feature, but that's the best match out of what is available in the towncrier configuration.

I wish we had a better name for documentation changes.

https://github.com/plone/plone.api/blob/master/pyproject.toml#L12-L16

Should I make a new issue to request that addition for a future PR, not this one?

**Review by stevepiercy — APPROVED** [review]

File: news/497.feature (hunk 1 above)
Comment (ksuess) [reply news/497.feature:1]: Thought the same.

Would be nice to have a towncrier news entry type '.doc'.
I think plone.releaser is the place to suggest an addition. See https://github.com/plone/plone.releaser/blob/master/ADD-A-NEWS-ITEM.rst

**Comment by ksuess** [comment]
Usually in Plone repos I do not merge my own PRs. Can you do that please, @mauritsvanrees 
You know what else is necessary for the big picture. Thank you!

File: news/497.feature (hunk 1 above)
Comment (mauritsvanrees) [reply news/497.feature:1]: Each project that uses `towncrier` has a `pyproject.toml` where the known suffixes are defined. For packages like `plone.api` from which we include the documentation on docs.plone.org, it makes sense to allow `.doc`. Maybe for all packages.
I will merge this PR, then copy the `pyproject.toml` from `plone.restapi` and fix the news snippet extension.

Comment (stevepiercy) [reply news/497.feature:1]: I would prefer `.documentation` because PyCharm and macOS attempt to open `.doc` in Word or other Office app.

Comment (mauritsvanrees) [reply news/497.feature:1]: done

**Pull request merged**